package.json

```diff
@@ -1,27 +1,36 @@
 {
-  "name": "dbload-example-app",
+  "name": "dbload-example-app2",
   "version": "1.0.0",
   "main": "server/server.js",
   "scripts": {
     "pretest": "jshint ."
   },
   "dependencies": {
-    "async": "^0.9.0",
+    "async": "^1.4.2",
     "compression": "^1.0.3",
+    "cors": "^2.5.2",
     "errorhandler": "^1.1.1",
-    "loopback": "^2.8.0",
-    "loopback-boot": "^2.4.0",
-    "loopback-datasource-juggler": "^2.7.0",
-    "restler": "^3.2.2",
-    "serve-favicon": "^2.0.1"
+    "fs": "0.0.2",
+    "loopback": "^2.14.0",
+    "loopback-boot": "^2.6.5",
+    "loopback-connector-postgresql": "^2.3.1",
+    "loopback-connector-sqlite3": "^1.0.0",
+    "loopback-datasource-juggler": "^2.19.0",
+    "memcached": "^2.1.0",
```

Comment:
"mssql" raw, not "mysql"

ok for riak and leveldown

understand about loopback connectors with individual apps, testing elsewhere is fair, except why then are you depending on two loopback connectors?

for loopback-connector-sqlite3, just use sqlite3 directly (your code would remain the same, you are pulling the sqlite3 constructor out of the loopback-connector-sqlite3).